src/integration-tests/integration-test.py

```diff
@@ -407,6 +429,10 @@ def launch_repro(self) -> None:
         has_cdb = bool(which("cdb.exe"))
         has_gdb = bool(which("gdb"))
         for job_id in self.successful_jobs:
+            if not TARGETS[self.target_jobs[job_id]].test_repro:
```

Comment:
I'm assuming this lookup is infallible in context.

Reply:
yep.